docs/components/SideBar/SideBar.jsx

```diff
@@ -20,6 +20,7 @@ export default function SideBar({className}) {
         <NavLink href="/design/typography">Typography</NavLink>
         <NavLink href="/design/sizing">Sizing</NavLink>
         <NavLink href="/design/page-layout">Page Layout</NavLink>
+        <NavLink href="/design/compound-form">Compound Form</NavLink>
```

Comment:
Gotcha. Should we do that now? Or maybe better to wait till we have more Layout guidelines written out?